charts/console/templates/_helpers.tpl

```diff
@@ -141,11 +224,106 @@ Those are warnings and not errors, they are only output in NOTES.txt
 {{- define "conduktor.validateValues" -}}
 {{- $messages := list -}}
 {{- $messages := append $messages (include "conduktor.validateValues.database" .) -}}
+{{- $messages := append $messages (include "conduktor.validateValues.monitoring" .) -}}
 {{- $messages := without $messages "" -}}
 {{- $message := join "\n" $messages -}}
 
 {{- if $message -}}
-{{-   printf "\n\nYOUR DEPLOYMENT MIGHT NOT WORK:\n\n%s" $message -}}
+{{- required (printf "\n\nYOUR DEPLOYMENT WILL NOT WORK:\n\n%s" $message) .Values.unknown -}}
+{{- end -}}
+{{- end -}}
+
+
+{{/*
+Return platform monitoring api poll rate for clusters. Default to 60s
+*/}}
+{{- define "conduktor.monitoring.clustersRefreshInterval" -}}
+{{- $refresh := "60" -}}
+{{- if .Values.config.monitoring -}}
+{{- $refresh := (default $refresh (index .Values "config" "monitoring" "clusters-refresh-interval")) }}
+{{- end -}}
+{{- printf "%s" $refresh -}}
+{{- end -}}
+
+{{/*
+Return platform monitoring cortex url. Like http://cortex.nsp.svc.cluster.local:9009/
+*/}}
+{{- define "conduktor.monitoring.cortexUrl" -}}
+{{- $url := printf "http://%s:%d/" (include "conduktor.platformCortex.serviceDomain" .) (.Values.platformCortex.service.ports.cortex | int) -}}
+{{- if .Values.config.monitoring -}}
+{{- $url := (default $url (index .Values "config" "monitoring" "cortex-url")) }}
+{{- end -}}
+{{- printf "%s" $url -}}
+{{- end -}}
+
+{{/*
+Return platform monitoring alertmanager url. Like http://cortex.nsp.svc.cluster.local:9010/
+*/}}
+{{- define "conduktor.monitoring.alertManagerUrl" -}}
+{{- $url := printf "http://%s:%d/" (include "conduktor.platformCortex.serviceDomain" .) (.Values.platformCortex.service.ports.alertmanager | int) -}}
+{{- if .Values.config.monitoring -}}
+{{- $url := (default $url (index .Values "config" "monitoring" "alert-manager-url")) }}
+{{- end -}}
+{{- printf "%s" $url -}}
 {{- end -}}
+
+{{/*
+Return platform monitoring callback url. Like http://platform.nsp.svc.cluster.local:8080/monitoring/api/
+This is used by alertmanager as a webhook to send alerts to the platform
+*/}}
+{{- define "conduktor.monitoring.callbackUrl" -}}
+{{- $url := printf "http://%s:%d/monitoring/api/" (include "conduktor.platform.serviceDomain" .) (.Values.service.ports.http | int) -}}
```

Comment:
I believe it might be missing TLS consideration (at pod level), but maybe it shouldn't be an issue as we got redirection from HTTP to HTTPS

Reply:
Maybe adding a cortex test with TLS at pod level could be useful to have

Reply:
Indeed, will add that. And it should already have a tests with cortex and platform in TLS, but our tests do not check more that healthcheck endpoint that I think don't take account platform<->cortex connectivity. Will check if I can come up with something.